eng/common/scripts/Verify-Links.ps1

```diff
@@ -13,6 +13,12 @@ param (
   [string] $rootUrl = "",
   # list of http status codes count as broken links. Defaults to 400, 401, 404, SocketError.HostNotFound = 11001, SocketError.NoData = 11004
   [array] $errorStatusCodes = @(400, 401, 404, 11001, 11004),
+  # flag to allow resolving relative paths or not
+  [bool] $resolveRelativeLinks = $true,
```

Comment:
Looks like this is not needed any longer as it was a bad merge. I'd actually suggest you put the checklinkGuidance parameter here instead of at the end of the argument list.

Reply:
I have another PR to address this. https://github.com/Azure/azure-sdk-tools/pull/895
Do we want to combine these two?

Reply:
The resolveRelativeLinks parameter is no longer used after you merged https://github.com/Azure/azure-sdk-tools/pull/865 so this looks like it is adding it back.

Reply:
Will remove it.